modules/services/nightly-test-runner.nix

```
@@ -49,8 +49,8 @@ let
     function run_and_rename {
       local NEW_RESULT_FILE="OK-$RESULT_FILE"
 
-      run-nightly-test | tee $RESULT_FILE
-      if [[ $? != 0 ]]; then
+      local status=$(run-nightly-test > $RESULT_FILE 2>&1)$?
```

This line is kind of weird. We're concatenating the output of `run-nightly-test > $RESULT_FILE 2>&1` with its status code. I guess we're just relying on its output being nothing, since we piped it into a file? Seems clearer to do something like this
```suggestion
      run-nightly-test > $RESULT_FILE 2>&1
      local status=$?
```